projects/app-host/src/app/pages/dynamic/dynamic.component.html

```diff
@@ -137,7 +137,7 @@ <h3 class="mat-display-5 plain-text whats-next-text">
           <lcu-setup-manage-element
             *ngIf="State && !State.Loading"
             [dashboard]="State.Dashboard"
-            [devices]="State.Devices"
+            [devices-config]="State.Devices"
```

Comment:
I am good with this rename to DevicesConfig, but to prevent misunderstandings, believe we should update it further:

`[devices-config]="State.DevicesConfig"`

So you would have to update the root property on the state in both TS and the State API